.github/actions/smoke-test/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+TEMPLATE_ID="$1"
+
+set -e
+
+shopt -s dotglob
+
+SRC_DIR="/tmp/${TEMPLATE_ID}"
```

Comment:
If a variable is going to be set once and then shouldn't be changed, please use `readonly`